common/models/user.js

```diff
@@ -573,6 +575,19 @@ module.exports = function(User) {
     this.settings.maxTTL = this.settings.maxTTL || DEFAULT_MAX_TTL;
     this.settings.ttl = this.settings.ttl || DEFAULT_TTL;
 
+    // user email case sensitivity default to true
+    if (this.settings.caseSensitiveEmail === undefined) {
```

Comment:
Actually, can't you just set this default to true somewhere before? ie. When configs are read in, etc.